elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -52,37 +53,52 @@ async def attach_cve_flaws_cli(runtime: Runtime, advisory_id: int, noop: bool, d
     runtime.logger.info("Getting advisory %s", advisory_id)
     advisory = Erratum(errata_id=advisory_id)
     exit_code = 0
-    for b in runtime.bug_trackers.values():
-        try:
-            await attach_cve_flaws(runtime, advisory_id, noop, advisory, b)
-        except Exception as e:
-            runtime.logger.error(traceback.format_exc())
-            runtime.logger.error(f'exception with {b.type} bug tracker: {e}')
-            exit_code = 1
+
+    # Flaw bugs associated with jira tracker bugs
+    # exist in bugzilla. so to work with jira trackers
+    # we need both bugzilla and jira instances initialized
+    if runtime.only_jira:
+        runtime.use_jira = True
+
+    tasks = []
+    for bug_tracker in runtime.bug_trackers.values():
+        flaw_bug_tracker = runtime.bug_trackers['bugzilla'] if bug_tracker.type == 'jira' else None
+        tasks.append(asyncio.create_task(get_flaws(runtime, advisory, bug_tracker,
+                                                   flaw_bug_tracker, noop)))
+    try:
+        lists_of_flaw_bugs = await asyncio.gather(*tasks)
+        flaw_bugs = sum(lists_of_flaw_bugs, [])
+        if flaw_bugs:
+            bug_tracker = runtime.bug_trackers['bugzilla']
+            _update(runtime, advisory, flaw_bugs, bug_tracker, noop)
+    except Exception as e:
+        runtime.logger.error(traceback.format_exc())
+        runtime.logger.error(f'Exception: {e}')
+        exit_code = 1
     sys.exit(exit_code)
 
 
-async def attach_cve_flaws(runtime, advisory_id, noop, advisory, bug_tracker):
+async def get_flaws(runtime, advisory, bug_tracker, flaw_bug_tracker, noop):
```

Comment:
Is this function become get_first_fix_flaw_bugs ?

Reply:
No I refactored the function to return flaw bugs and then at the end attach all the flaw bugs